### ISSUE TITLE
Local Adversarial loss

### ISSUE THREAD

**Issue by islamtashfiq** [carpedm20/simulated-unsupervised-tensorflow#2]
Hi,

In the paper the authors mentioned about local adversarial loss and the fact that the discriminator is a fully convolutional, however I couldn't find that part in your code. Could you point me to that.

Thanks

**Comment by carpedm20** [comment]
Local adversarial loss is implemented like [this](https://github.com/carpedm20/simulated-unsupervised-tensorflow/blob/master/model.py#L133) and the cross-entropy loss values is summed over local patches. Fully convolutional discriminator is implemented like [this](https://github.com/carpedm20/simulated-unsupervised-tensorflow/blob/master/model.py#L302).
